templates/web/base/admin/contact-form.html

```diff
@@ -41,21 +41,19 @@
         [% loc("Check <strong>confirmed</strong> to indicate that this contact has been confirmed as correct.
                 If you are not sure of the origin or validity of the contact, leave this unchecked.") %]
     </p>
-  </div>
-  <p>
-    <input type="checkbox" name="confirmed" value="1" id="confirmed"[% ' checked' IF contact.confirmed OR contact.confirmed == "" %]>
-    <label for="confirmed" class="inline">[% loc('Confirmed') %]</label>
-  </p>
-
-  <div class="admin-hint">
     <p>
       [% loc("Check <strong>deleted</strong> to remove the category from use. 
         It will not appear as an available category in the drop-down menu on the report-a-problem page.") %]
     </p>
   </div>
   <p>
-    <input type="checkbox" name="deleted" value="1" id="deleted"[% ' checked' IF contact.deleted %]>
-    <label for="deleted" class="inline">[% loc('Deleted') %]</label>
+    <label for="state">[% loc('State') %]</label>
```

Comment:
The tooltip wording doesn't quite match the UI any more, could this be updated to describe the new 'inactive' behaviour?